src/components/app/ProfileViewer.css

```diff
@@ -38,11 +38,19 @@
 }
 
 .profileViewer {
+  flex-flow: column nowrap;
   animation-duration: 200ms;
   animation-name: profileViewerFadeIn;
   background-color: #fff;
 }
 
+body,
```

Comment:
body doesn't belong to ProfileViewer component. Can you please move this back to style.css? It's fine to duplicate these rules.

Reply:
You're right, sure.